src/sentry/api/endpoints/organization_events_v2.py

```diff
@@ -0,0 +1,33 @@
+from __future__ import absolute_import
+
+from functools import partial
+
+from rest_framework.response import Response
+
+from sentry.api.bases import OrganizationEventsV2EndpointBase, OrganizationEventsError, NoProjects
+from sentry.api.paginator import GenericOffsetPaginator
+from sentry.utils.snuba import transform_aliases_and_query
+
+
+class OrganizationEventsV2Endpoint(OrganizationEventsV2EndpointBase):
```

Comment:
Need to add this to urls.py

Reply:
The idea behind this (and not redirecting below) is that this API replaces the old one if the user has the flag switched on. We don't want to create a new endpoint since this will be backwards compatible with the existing events API.

Reply:
In that case, does it make sense to just have a separate method in the existing endpoint that we call when they have the new feature?

Reply:
I moved this method into the same class so this is clearer, keeping test files separate for now so they don't get excessively long